withings_sync/garmin.py

```diff
@@ -22,31 +21,34 @@ class APIException(Exception):
 class GarminConnect:
     """Main GarminConnect class"""
 
-    UPLOAD_URL = "https://connect.garmin.com/upload-service/upload/.fit"
-
-    # From https://github.com/cpfair/tapiriik
     @staticmethod
```

Comment:
This question should be in the MR before. However, is this method static anymore? We reference garth.* stuff.
If yes, my bad im still kind of beginner in python/pep's.